lighthouse-core/config/default-config.js

```diff
@@ -452,6 +453,7 @@ const defaultConfig = {
         {id: 'duplicated-javascript', weight: 0, group: 'load-opportunities'},
         {id: 'legacy-javascript', weight: 0, group: 'load-opportunities'},
         {id: 'preload-lcp-image', weight: 0, group: 'load-opportunities'},
+        {id: 'lcp-lazy-loaded', weight: 0, group: 'load-opportunities'},
```

Comment:
we only put an audit into `load-opportunities` today if we can quantify the impact on resources, for now let's leave it as `diagnostics`